src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -106,15 +116,37 @@
 	private FrameLayout mLeftPane;
 	private FrameLayout mRightPane;
 
+	private TabLayout mTabLayout;
+
 	private boolean isMenuShown = true;
 
 	private SharedPreferences sharedPreferences;
+	private LinearLayout userInterface;
+	private DrawerLayout mDrawer;
+	private int activeTab = 0;
+	private PostSort mOrder = PostSort.HOT;
+	private ActionBar actionbar;
 
 	@Override
 	protected boolean baseActivityIsActionBarBackEnabled() {
 		return false;
 	}
 
+	@Override
+	protected void onResume() {
+		super.onResume();
+		doRefresh(RefreshableFragment.MAIN_RELAYOUT, false, null);
+		if(PrefsUtility.appearance_navigation_type(this, sharedPreferences).equals("drawer_tabs") && !twoPane) {
+			mDrawer.openDrawer(GravityCompat.START, true);
```

Comment:
Does this open and close the drawer every time the activity gets resumed? I think that could be quite distracting/confusing for the user.